apps/prairielearn/src/admin_queries/course_instance_active_enrollments.ts

```diff
@@ -1,4 +1,30 @@
-import { runLegacySqlAdminQuery } from './util.js';
+import { type AdministratorQuerySpecs, runLegacySqlAdminQuery } from './lib/util.js';
+
+export const specs: AdministratorQuerySpecs = {
+  description: 'Course instances that are active within a given date range.',
+  params: [
+    {
+      name: 'institution_short_name',
+      description: 'The short name of the institution (required).',
+      default: '',
+    },
+    {
+      name: 'start_date',
+      description: 'Start of the date range (e.g., first day of the academic year).',
+      default: '2021-01-01T00:00:00',
```

Comment:
Moving this to TS means that we could potentially have the default be a dynamic value (e.g., now/today, or a value computed from it). Not sure if it's worth the effort.